eox_core/api/v1/tests/test_users.py

```diff
@@ -78,68 +160,216 @@ def test_update_role_not_allowed(self, get_edxapp_user, signup_source, _):
         self.assertEqual(response.content, '{"detail":["You can\'t update users with roles like staff or superuser."]}'
                          .encode())
 
+    @patch_permissions
+    @patch('eox_core.api.v1.serializers.UserSignupSource')
+    @patch('eox_core.api.v1.views.get_edxapp_user')
+    @patch('eox_core.api.v1.views.EdxappUserReadOnlySerializer')
+    def test_update_extra_registration_fields_success(self, user_serializer, get_edxapp_user, signup_source, _):
+        """Used to test updating safe extra fields of an edxapp user."""
+        update_data = {
+            "email": "test",
+            "fullname": "test",
+            "country": "gr",
+            "goals": "new goals",
+        }
+        user_serializer.return_value.data = {}
+        get_edxapp_user.return_value = self.user
+        signup_source.objects.filter.return_value.count.return_value = 1
+
+        response = self.client.patch(self.url, data=update_data, format="json")
 
-@override_settings(extended_profile_fields=[
-    "father_last_name",
-    "mother_last_name",
-    "type_document",
-    "personal_id",
-    "mobile",
-    "address",
-])
-@override_settings(EDNX_CUSTOM_REGISTRATION_FIELDS=[
-    {
-        "label": "Father last name",
-        "name": "father_last_name",
-        "type": "text",
-    },
-    {
-        "label": "Mother last name",
-        "name": "mother_last_name",
-        "type": "text",
-    },
-    {
-        "label": "Document type",
-        "name": "type_document",
-        "options": [
-            "dni",
-            "passport",
-            "license",
-        ],
-        "type": "select",
-        "default": "dni",
-    },
-    {
-        "label": "Document number",
-        "name": "personal_id",
-        "type": "text",
-    },
-    {
-        "label": "Mobile",
-        "name": "mobile",
-        "type": "text",
-        "restrictions": {
-            "max_length": 12,
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
```

Comment:
you can also check the update (?)

Reply:
@mariajgrimaldi what do you mean? 
I don't think it's really possible to check the update since the UserProfile is a model defined in the edx-platform. Here I'm just checking the request does not return any Validation error when trying to update the fields